messaging/kafka-streams-reactive-messaging/src/test/java/io/quarkus/ts/messaging/kafka/DevModeRedPandaDevServiceUserExperienceIT.java

```diff
@@ -15,7 +15,7 @@
 @QuarkusScenario
 public class DevModeRedPandaDevServiceUserExperienceIT {
 
-    private static final String RED_PANDA_VERSION = "latest";
+    private static final String RED_PANDA_VERSION = "v21.10.3";
```

Comment:
We are checking just logs, not concerned about the latest docker image being used